model/src/gateway/payload/message_delete_bulk.rs

```diff
@@ -1,8 +1,9 @@
-use crate::id::{ChannelId, MessageId};
+use crate::id::{ChannelId, GuildId, MessageId};
 use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
 pub struct MessageDeleteBulk {
     pub channel_id: ChannelId,
+    pub guild_id: Option<GuildId>,
```

Comment:
I was going to say that since this event only happens in guilds it shouldn't be optional, but [the docs](https://discord.com/developers/docs/topics/gateway#message-delete-bulk-message-delete-bulk-event-fields) say it is... huh.

Reply:
I guess this is a case of if Twilight wants to be 100% docs compliant or make a possibly smarter choice based off a [really vague comment that might not even be serious](https://github.com/discord/discord-api-docs/issues/1633#issuecomment-629501625). From the API usage front, most end users will likely just `.unwrap()` this field anyway.

Reply:
I generally think it is best to just follow the docs, while I agree that it is annoying.

Reply:
Not my preference, but fair enough. Resolving this comment.